starter-core/src/main/java/io/micronaut/starter/feature/other/ProjectLombok.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.other;
+
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.feature.Category;
+import io.micronaut.starter.feature.LanguageSpecificFeature;
+import io.micronaut.starter.options.BuildTool;
+import io.micronaut.starter.options.Language;
+
+import javax.inject.Singleton;
+
+@Singleton
+public class ProjectLombok implements LanguageSpecificFeature {
+
+    private static final String LOMBOK_MAVEN_KEY = "lombok.version";
+    private static final String LOMBOK_GRADLE_KEY = "lombokVersion";
+    private static final String LOMBOK_VERSION = "1.18.16";
```

Comment:
Please move this hard coded version to the BOM and call `VersionInfo.getDependencyVersion(String id)` instead

Reply:
@graemerocher This is another thing like testcontainers. I'd like to get group consensus before we add this to the bom